src/main/java/school/lemon/changerequest/java/introduction/hw2/Task1.java

```diff
@@ -2,6 +2,9 @@
 
 public class Task1 {
     public static String makeTags(String tag, String text) {
-        return "";
+
+        String result = "<" + tag + ">" + text + "</" + tag + ">";
```

Comment:
For sure this solution will work, but in Java strings are immutable and if we don't rely on compiler optimizations,  your code will create 7 String objects.

It's better to use `String.format` if you have pre-defined format, of `StringBuilder` if you gonna create `dynamic` strings.